src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -314,4 +315,12 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
         log.debug("Notification scheduled successfully");
     }
 
+    @Override
+    public void handleKedaInvocationForMediaRequests() {
+        Optional<MediaRequestEntity> openAudioRequests = mediaRequestService.getOldestMediaRequestByStatus(AudioRequestStatus.OPEN);
+        if (openAudioRequests.isPresent()) {
+            processAudioRequest(openAudioRequests.get().getId());
```

Comment:
Before calling processAudioRequest, we will have to update the media_request table record with PROCESSING status.

Reply:
The existing `processAudioRequest()` method already sets status to PROCESSING, so I don't think we need to do this here.

Snippet:
```java
    public UUID processAudioRequest(Integer requestId) {

        log.debug("Starting processing for audio request id: {}", requestId);
        mediaRequestService.updateAudioRequestStatus(requestId, PROCESSING);
        ...etc
}
```